src/order/order_iso_nat.lean

```diff
@@ -7,12 +7,25 @@ import data.equiv.denumerable
 import order.preorder_hom
 import order.conditionally_complete_lattice
 
+/-!
+# Relation embeddings from the naturals
+
+This file allows translation from monotone functions `ℕ → α` to order embeddings `ℕ ↪ α` and
+defines the limit value of an eventually-constant sequence.
+
+## Main declarations
+
+* `nat_<lt/gt>`: Make an order embedding `ℕ ↪ α` from an increasing/decreasing function `ℕ → α`.
```

Comment:
```suggestion
* `nat_lt`/`nat_gt`: Make an order embedding `ℕ ↪ α` from an increasing/decreasing function `ℕ → α`.
```
so that docgen creates links

Reply:
Oh wait, doc-gen creates links? Then what do you think of the similar thing I did in #7799?

Reply:
I think it would be better to avoid it!